dev/tools/gen_defaults/lib/chip_template.dart

```diff
@@ -10,7 +10,7 @@ class ChipTemplate extends TokenTemplate {
     super.textThemePrefix = '_textTheme.'
   });
 
-  static const String tokenGroup = 'md.comp.assist-chip';
+  static const String tokenGroup = 'md.comp.filter-chip';
```

Comment:
Based on the comment in the original issue, this change makes sense to me. Maybe @TahaTesser has more context about this:)

Reply:
Since `chip_template` is for generic chip, using filter chip tokens to get most tokens available makes sense.